paasta_tools/cli/cmds/status.py

```diff
@@ -1124,6 +1132,112 @@ def print_flink_status(
     return 0
 
 
+def print_kubernetes_status_v2(
+    cluster: str,
+    service: str,
+    instance: str,
+    output: List[str],
+    status,
+    verbose: int = 0,
+) -> int:
+    instance_state = get_instance_state(status)
+    output.append(f"    State: {instance_state}")
+    output.append("    Running versions:")
+    output.append("      " + PaastaColors.green("Rerun with -v to see all replicas"))
+    output.extend([f"      {line}" for line in get_versions_table(status.replicasets)])
+    if status.error_message:
+        output.append("    " + PaastaColors.red(status.error_message))
+        return 1
+    else:
+        return 0
+
+
+def get_instance_state(status) -> str:
+    num_replicasets = len(status.replicasets)
+    num_ready_replicas = sum(r.ready_replicas for r in status.replicasets)
+    if status.desired_state == "stop":
+        if num_replicasets == 0:
+            return PaastaColors.red("Stopped")
```

Comment:
All the states should organized in a class somewhere or put as global constants somewhere for consistency.

Reply:
yeah can do

Reply:
actually I need to get out of here before PTO, so I left a TODO, sorry :-/